app/views/content_items/publication.html.erb

```diff
@@ -53,8 +59,15 @@
       <%= render 'components/published-dates', {
         published: @content_item.published,
         last_updated: @content_item.updated,
-        history: @content_item.history
+        history: @content_item.history,
+        margin_bottom: @notification_button_visible ? 3 : nil
```

Comment:
Am I right in that this is the section at the bottom of the page? I've just been playing with the margin bottom and it seems that this check isn't necessary - if the notification button isn't present, having this margin bottom or not on the published dates component doesn't change the layout anyway, as the margin is the same as the margin already applied by the wrapping `responsive-bottom-margin` element.

Reply:
Yeah you're correct about that, good spot. Will update